doc/contributing.rst

```diff
@@ -7,23 +7,36 @@ call functions in the Stan C++ library's ``stan::services`` namespace.
 **httpstan** was originally developed as a "backend" for a Stan interface
 written in Python, PyStan_.
 
-httpstan emerged out of a rewrite of an existing piece of software (PyStan)
-which was difficult to maintain. httpstan aims to be maintainable. Code
-contributions should be readable and easy to understand.
+Goals:
+
+- Provide access to frequently-used functions in Stan C++ library's ``stan::services`` namespace.
+- Maintainability.
+
+Non-goals:
+
+- Provide access to functions other than those in the ``stan::services`` namespace.
+  There are only three exceptions: ``model_base_crtp.log_prob``,
+  ``model_base_crtp.write_array``, and ``stan::model::log_prob_grad``.
+
+If these goals and non-goals strike you as restrictive, remember that httpstan
+is open source software which you are free to fork and customize.
```

Comment:
Is there a way to soften the tone of this sentence.

There might be a valid case for other functionality, and we just want kindly to remind users that people are free to fork and customize package if needed.